tests/e2e/login/privacy-policy.spec.js

```diff
@@ -161,6 +161,7 @@ describe('Privacy policy', () => {
       // Update privacy policies
       const newPolicyText = 'Cette text est totalement different c`est fois!';
       await privacyPolicyPage.updatePrivacyPolicy('privacy-policies', 'fr_attachment', newPolicyText);
+      await utils.closeTour();
```

Comment:
The tour is controlled by a param on the user doc called `known`. If you update the offlineUser  object starting on line 14 it should not display.  This means we don't have to have the same code everywhere and not depending on closing the modal all the time.  We should do this for any user we create and if you use the `user-settings` factory it is defaulted. Then we can create a set of tests that specifically test the tour and all our other tests won't get interrupted by it. 

 ``` const offlineUser = {
    known: true,
    username: 'offline',
    password: password,
    place: {
      _id: 'fixture:offline',
      type: 'health_center',
      name: 'offline',
      parent: 'PARENT_PLACE'
    },
    ```